OpenRA.Game/HotkeyDefinition.cs

```diff
@@ -20,6 +20,7 @@ public sealed class HotkeyDefinition
 		public readonly Hotkey Default = Hotkey.Invalid;
 		public readonly string Description = "";
 		public readonly HashSet<string> Types = new HashSet<string>();
+		public bool HasDuplicates = false;
```

Comment:
This really needs to be set as part of the `HotkeyManager` parsing! In principle any part of the game code could query this, and will read a wrong / uninitialized value.

Reply:
I see. I will file a follow up.